continuous_integration/appveyor/setup_conda_environment.cmd

```diff
@@ -26,7 +26,8 @@ copy NUL %CONDA_PREFIX%\conda-meta\pinned
 @rem Install optional dependencies for tests
 %CONDA_INSTALL% numpy pandas cloudpickle distributed
 %CONDA_INSTALL% bcolz bokeh h5py ipython lz4 psutil pytables s3fs scipy
-%CONDA_INSTALL% -c conda-forge fastparquet snappy
+%CONDA_INSTALL% -c conda-forge fastparquet snappy zarr
```

Comment:
python-snappy ?

Reply:
This seems to be the last remaining comment on this PR. @mpeaton would you mind updating this to use `python-snappy` instead of `snappy`? (`python-snappy` will install the `snappy` library along with Python bindings)